source/developers-guide/attribute-system/index.md

```diff
@@ -652,3 +657,99 @@ Ext.define('Shopware.form.field.SwagAttributeGrid', {
     }
 });
 ```
+
+## Move attribute fields into another form
+
+All attribute fields are place in a new fieldset or tab, which sometimes doesn't fit your needs. The following example will show you how to move an article attribute field out of the fieldset into the basic information fieldset.
+
+This section presumes, that you already have your own attribute created and registered your view directory.
```

Comment:
This section presumes, that you already created your own attribute and registered your view directory.